source/api/image/3.0/change-log.md

```diff
@@ -68,9 +68,11 @@ The `license` property was renamed to `rights` to accommodate both rights statem
 
 See issues [#644](https://github.com/IIIF/api/issues/644) and [#1479](https://github.com/IIIF/api/issues/1479).
 
-#### 1.2.4. Require the `type` property on all resources, with new values
+#### 1.2.4. Require the `type` property on info.json and external references, with new values
```

Comment:
Should `info.json` be code font? Not a strong opinion but I lean that way

Reply:
Not in a title, I think?